tests/app/helpers/test_template_helpers.py

```diff
@@ -103,86 +103,115 @@ def test_footer_context_census_nisra_theme(app: Flask, expected_footer_nisra_the
     assert result == expected_footer_nisra_theme
 
 
-def test_get_page_header_context_business(app: Flask):
-    expected = {
-        "orgLogo": "ons-logo-en",
-        "orgLogoAlt": "Office for National Statistics logo",
-    }
-
-    with app.app_context():
-        survey_config = SurveyConfig()
-
-        result = ContextHelper(
-            language="en",
-            is_post_submission=False,
-            include_csrf_token=True,
-            survey_config=survey_config,
-        ).context["page_header"]
-
-    assert result == expected
-
-
-def test_get_page_header_context_social(app: Flask):
-    expected = {
-        "orgLogo": "ons-logo-en",
-        "orgLogoAlt": "Office for National Statistics logo",
-        "title": "ONS Social Surveys",
-    }
-
-    with app.app_context():
-        survey_config = SocialSurveyConfig()
-
-        result = ContextHelper(
-            language="en",
-            is_post_submission=False,
-            include_csrf_token=True,
-            survey_config=survey_config,
-        ).context["page_header"]
-
-    assert result == expected
-
-
-def test_get_page_header_context_census(app: Flask):
-    expected = {
-        "title": "Census 2021",
-        "orgLogo": "ons-logo-en",
-        "orgLogoAlt": "Office for National Statistics logo",
-        "titleLogo": "census-logo-en",
-        "titleLogoAlt": "Census 2021",
-    }
-
-    with app.app_context():
-        survey_config = CensusSurveyConfig()
-
-        result = ContextHelper(
-            language="en",
-            is_post_submission=False,
-            include_csrf_token=True,
-            survey_config=survey_config,
-        ).context["page_header"]
-
-    assert result == expected
-
-
-def test_get_page_header_context_census_nisra(app: Flask):
-    expected = {
-        "title": "Census 2021",
-        "orgLogo": "nisra-logo",
-        "orgLogoAlt": "Northern Ireland Statistics and Research Agency logo",
-        "titleLogo": "census-logo-en",
-        "titleLogoAlt": "Census 2021",
-        "customHeaderLogo": True,
-        "orgMobileLogo": "nisra-logo-mobile",
-    }
-
+@pytest.mark.parametrize(
+    "theme, survey_config, expected",
+    (
+        (
+            SurveyType.BUSINESS,
+            BusinessSurveyConfig(),
+            {
+                "orgLogo": "ons-logo-en",
+                "orgLogoAlt": "Office for National Statistics logo",
+                "title": "ONS Business Surveys",
+            },
+        ),
```

Comment:
We need a test that show a non default title working. All these tests asserts the defaults only. If the cookie session has a `survey_title`, then that is what should be rendered.

Reply:
I've added couple variants with `'survey_title` in cookie present, should cover the scenarios.